embassy-stm32/src/usb/otg.rs

```diff
@@ -311,6 +314,26 @@ impl<'d, T: Instance> Bus<'d, T> {
             });
         });
 
+        #[cfg(all(stm32u5, peri_usb_otg_hs))]
+        {
+            crate::pac::RCC.ccipr2().modify(|w| {
+                w.set_otghssel(Otghssel::PLL1_P);
```

Comment:
this overrides the user's choice in `config.rcc.mux.otghssel`. you shouldn't write to it explicitly, rcc init already does with the value the user chose in `config.rcc.mux.otghssel`.

Reply:
Whoops. That was meant to have gone. Will remove it.